app/types/GlobalState.ts

```diff
@@ -0,0 +1 @@
+export type updateUserKey = 'firstname' | 'lastname' | 'bio' | 'username' | 'timezone';
```

Comment:
Nitpick: 
Types are generally defined with the first character capitalized.

Reply:
thanks